airflow/config_templates/default_airflow.cfg

```diff
@@ -464,11 +464,6 @@ dag_dir_list_interval = 300
 # How often should stats be printed to the logs
 print_stats_interval = 30
 
-# If the last scheduler heartbeat happened more than scheduler_health_check_threshold ago (in seconds),
-# scheduler is considered unhealthy.
-# This is used by the health check in the "/health" endpoint
-scheduler_health_check_threshold = 30
```

Comment:
Does anyone thing we need an UPDATING note for this?

Should I replace it with something that controls the grace_multiplier for SchedulerJob? (or keep this setting and have SchedulerJob use a fixed timeout instead) @feluelle @XD-DENG thoughts?

Reply:
@ashb , I would suggest to keep this setting and let SchedulerJob use it as a fixed heartbeat timeout. Number `2.1` here is just too confusing to me.

Reply:
> Does anyone thing we need an UPDATING note for this?

I think it is not necessary, but please don't forget to update the docs accordingly to your changes: https://github.com/apache/airflow/blob/master/docs/howto/check-health.rst

> Should I replace it with something that controls the grace_multiplier for SchedulerJob? (or keep this setting and have SchedulerJob use a fixed timeout instead) @feluelle @XD-DENG thoughts?

I prefer a fixed value over a multiplier, because it is clearer to me. But I would also be fine with having a grace_multiplier as long as it is documented properly :)

Reply:
I've kept this and sub-classed is_alive for SchedulerJob.